README.rst

```diff
@@ -20,33 +20,68 @@ configurations remain synchronized with the state of all databases in the
 cluster. PgQ is used for buffering mutation records until they are collected by
 the relay and forwarded to their final destination.
 
+Function
+========
+
+``pgshovel`` is a system which replicates mutations (``INSERT``, ``UPDATE`` and ``DELETE`` operations) made to a PostgreSQL database out of the database and into an ordered stream of transaction-aware mutation messages. These messages can then be consumed by 3rd party systems as needed. Commonly, the mutation stream is consumed by data warehouse operations, cache updating services, search index services or can even be processed into other new streams.
+
+At Disqus ``pgshovel`` is actively being developed to replicate changes from our master PostgreSQLdatabase - which contains the core commenting and other relational data - into our data warehousing system. There is further planned work to use the mutation stream to power a denormalized objects database, offering highly available low-latency access to object data.
+
 Concepts
 ========
 
+``pgshovel`` has a few core concepts which are important to understanding it:
+
 Cluster
 -------
 
-A cluster contains replication sets.
+A cluster contains a collection of replication sets.
 
 Replication Set
 ---------------
 
-Replication sets contain a collection of tables that are replicated together.
+A replication set is configuration defining a collection of tables which have their mutations replicated together.
```

Comment:
> I'm not sure if this could use more elaboration on the definition of "consistency" (preserves referential integrity, etc.)

Ha ha - yeah explaining this has been hard.

> I'm also not sure if "can be" is the right wording, "are" might be better there.

Yeah "are" is better.  Will change.